kmir/src/tests/integration/data/exec-smir/arithmetic/arithmetic-unchecked-runs.state

```diff
@@ -34,21 +34,21 @@
       ListItem ( newLocal ( ty ( 1 ) , mutabilityMut ) )
       ListItem ( typedValue ( Integer ( 254 , 8 , false ) , ty ( 9 ) , mutabilityNot ) )
       ListItem ( typedValue ( Aggregate ( variantIdx ( 0 ) , ListItem ( Moved )
-      ListItem ( Moved ) ) , ty ( 28 ) , mutabilityMut ) )
+      ListItem ( Moved ) ) , TyUnknown , mutabilityMut ) )
       ListItem ( typedValue ( Integer ( 255 , 8 , false ) , ty ( 9 ) , mutabilityNot ) )
       ListItem ( typedValue ( Integer ( -100 , 8 , true ) , ty ( 2 ) , mutabilityNot ) )
       ListItem ( typedValue ( Aggregate ( variantIdx ( 0 ) , ListItem ( Moved )
-      ListItem ( Moved ) ) , ty ( 26 ) , mutabilityMut ) )
+      ListItem ( Moved ) ) , TyUnknown , mutabilityMut ) )
       ListItem ( typedValue ( Integer ( -128 , 8 , true ) , ty ( 2 ) , mutabilityNot ) )
       ListItem ( typedValue ( Integer ( -32640 , 16 , true ) , ty ( 35 ) , mutabilityNot ) )
-      ListItem ( typedValue ( Integer ( 255 , 16 , true ) , ty ( 35 ) , mutabilityMut ) )
-      ListItem ( typedValue ( Integer ( -128 , 16 , true ) , ty ( 35 ) , mutabilityMut ) )
+      ListItem ( typedValue ( Integer ( 255 , 16 , true ) , ty ( 35 ) , mutabilityNot ) )
+      ListItem ( typedValue ( Integer ( -128 , 16 , true ) , ty ( 35 ) , mutabilityNot ) )
```

Comment:
I believe if we restore the `#setLocalValue` behaviour of keeping `Ty` and `Mutability` of the written local these tests can pass unmodified.